presto-main/src/main/java/io/prestosql/server/PrestoSystemRequirements.java

```diff
@@ -96,7 +99,14 @@ private static void verifyJavaVersion()
         }
 
         JavaVersion version = JavaVersion.parse(javaVersion);
-        if (version.getMajor() == 8 && version.getUpdate().isPresent() && version.getUpdate().getAsInt() >= 161) {
+
+        if (version.getMajor() >= 11) {
```

Comment:
That seems risky given that only 11 is LTS and next LTS is 17 .. and there will be all sorts of changes until then. It might be worth throwing warning that the default supported release is 11 and your mileage may vary with newer releases ..

Reply:
We already support versions higher than 11, although we recommend people use 11 at this point. We could add a warning for that, too, but it'd be a separate change.

The motivation behind this one is to start the process of moving development away from Java 8 and to Java 11.

Reply:
Cool.